Kconfig.dependecies

```
@@ -211,24 +211,17 @@ config SIDEWALK_DFU_SERVICE_BLE_DEFAULTS
 	imply MCUMGR
 	imply MCUMGR_GRP_IMG
 	imply MCUMGR_GRP_OS
+	imply MCUMGR_MGMT_NOTIFICATION_HOOKS
+	imply MCUMGR_GRP_IMG_UPLOAD_CHECK_HOOK
+	imply MCUMGR_GRP_IMG_STATUS_HOOKS
+	imply MCUMGR_GRP_OS_RESET_HOOK
+	imply MCUMGR_SMP_COMMAND_STATUS_HOOKS
 	imply MCUMGR_TRANSPORT_BT
 	imply MCUMGR_SMP_BT_CONN_PARAM_CONTROL
```

Not related with this PR, but this config changes the name in NCS
```suggestion
	imply MCUMGR_TRANSPORT_BT_CONN_PARAM_CONTROL
```

---

done